src/pages/_app.tsx

```diff
@@ -10,37 +9,6 @@ import { MakairaAppProvider } from '@/makaira/MakairaAppProvider'
 import '@/styles/mixins.scss'
 import '@/styles/globals.scss'
 
-const futura = localFont({
```

Comment:
Why are your removing the local front from Next here? I don't see a reason for getting rid of it so please change it back.

Reply:
rc-component popups are always added outside of the <main> (Ex: select options panel, dropdown menu). Do you have any suggestions to add the fonts to the body? I temporary added the fonts in global scss.

Reply:
I think you can set the font with the provided variable of Next.js.
`font-family: var(--font-family-regular)`

If this doesn't work than keep  than you can keep the font definition in the global scss.